microsoft-teams/1.0/modules/ROOT/pages/microsoft-teams-connector-examples.adoc

```diff
@@ -0,0 +1,427 @@
+. Select *Connector Configuration > Microsoft Teams Authorization Code Config* and click *OK*.
+. Enter the following values:
++
+[%header%autowidth.spread]
+|===
+| Field | Value
+| Name | `OAuth_Authorization_Code_Config`
+| Consumer key | `${config.oauth.authorization.code.consumerKey}`
+| Consumer secret | `${config.oauth.authorization.code.consumerSecret}`
+| Authorization url | `${config.oauth.authorization.code.authorizationUrl}`
+| Access token url |`${config.oauth.authorization.code.accessTokenUrl}`
+| Scopes | Leave the default: `+https://graph.microsoft.com/.default+`
+| Listener config | `auth`
+| Callback path | `/callback`
+| Authorize path | `authorizePath`
+| External callback url | `+http://localhost:8083/callback+`
+|===
+. Click *OK*.
+. Click the *Global Elements* tab and click *Create*.
+. Select *Connector Configuration > Microsoft Teams Client Credentials Config* and click *OK*.
+. Enter the following values:
++
+[%header%autowidth.spread]
+|===
+| Field | Value
+| Name | `Microsoft_Teams_Client_Credentials_Config`
+| Client id |  `${config.oauth.client.credentials.clientId}`
+| Client secret | `${config.oauth.client.credentials.clientSecret}`
+| Token url | `${config.oauth.client.credentials.tokenUrl}`
+| Scopes | Leave the default: `+https://graph.microsoft.com/.default+`
+|===
+. Click *OK*.
+
+=== Configure a Global Element for the Properties File
+
+Configure a global element for the `mule-app.properties` file so that Mule knows where to find it:
+
+. Click the *Global Elements* tab and click *Create*.
+. In the *Choose Global Type* dialog, select *Configuration properties* and click *OK*.
+. In the *File* field, enter `mule.app.properties`.
+. Click *OK*.
+
+[[create-new-team]]
+== Create a New Team
+
+The following screenshot shows the Studio app flow for creating a new team:
+
+.Use a flow like this one to create a new team.
+image::ms-teams-create-team.png[Create a new team flow]
+
+Creating a new team involves configuring an HTTP *Listener* component, *Transform Message* component, *Create Team* operation, and second *Transform Message* component.
+
+To create the flow:
+
+. Create a new Mule project in Studio.
+. In the Mule Palette view, search for *HTTP* and select the *Listener* operation:
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. Accept the defaults.
+. In the HTTP properties window, set the *Path* field to `/createTeam`.
+
+=== Add the first Transform Message Component
+
+The first *Transform Message* component creates a template for the input used to create the team:
+
+. In the Mule Palette view, search for *transform message*:
+. Drag the *Transform Message* component onto the canvas, to the right of the *Listener* component.
+. In the *Transform Message* configuration, overlay the brackets in the *Output* section with this XML:
```

Comment:
Is there a reason to use "overlay" rather than replace? This is the first time that I have seen this phrasing in our docs is the reason I ask.